plaidml/bridge/keras/backend_test.py

```diff
@@ -1330,17 +1330,17 @@ def testRandomChanges(self, b):
     # sufficient. It may be worthwhile to make this test more resilient
     # to refactoring and make it test that the update portion is working
     # as expected.
-    @compareForwardClose(skip_tensorflow=True)
+    @compareForwardClose(skip_tensorflow=True, skip_theano=False)
     def testMovingAverageUpdate(self, b):
         return b.moving_average_update(b.variable(m(5, 4, 9, 3, 2)), b.variable(n(5, 4, 9, 3, 2)),
                                        0.95)[1]
 
-    @compareForwardClose(skip_tensorflow=True, atol=1e-6)
+    @compareForwardClose(skip_tensorflow=True, skip_theano=False, atol=1e-6)
     def testBatchNormAndUpdate(self, b):
         b.set_learning_phase(1)
         x = b.variable(n(4, 7))
-        moving_mean = b.variable(m(4, 1))
-        moving_var = b.variable(m(4, 1))
+        moving_mean = b.variable(m(4, 4))
+        moving_var = b.variable(m(4, 4))
```

Comment:
This is not a fix for this PR, but:

It looks like the weird shape (4, 4) is being introduced by the interaction of the lines
```
    # squeeze the mean and variance in all cases, that's what TF does
    m = squeeze(m)
    v = squeeze(v)
```
with the unsqueezed sizes (4, 1) for the moving mean & var; this ends up adding a (4, 1) tensor to a (4) tensor, which broadcasts to (4, 4). Probably we should switch to using size (4) for the moving mean & var

Reply:
OK, thanks for looking into this